assessment/serializers.py

```diff
@@ -47,12 +51,51 @@ class Meta:
             'is_private',
             'start_date_time',
             'end_date_time',
-            'admin',
-            'users',
             'translations',
             'questions',
+            'is_admin',
+            'in_users',
+            'name',
+            'description',
+
         )
 
+    def get_is_admin(self, obj):
+        """
+        The application should never directly report who the admin is.
+        This method reports if the current authenticated user is the admin.
+        """
+        try:
+            user = self.context.get('request').user
+        except Exception:
+            # raise serializers.ValidationError('Could not access request.user')
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=python%3AS125)